NewForce-Cohort-4/react-nutshell-spidereye
# Jp articles

## What is it?
- This component contains an article list that displays a list of cards containing the article title, url, and synopsis. The articles are sorted by date in descending order and each card has an edit and delete button. The articles displayed in the DOM are only for the user currently logged in.

## How to test it
- Test this component by registering as a user and logging in. Click the "Articles" link in the navbar and then click the "Add new article" button at the top of the page. You will be presented with a form that allows you to input the article title, URL, and synopsis. After you save the article it will be given a time stamp that allows us to sort the articles. After clicking the "Save" button you should see the article appear in the DOM. Repeat this several times to see how the articles are sorted in descending order by date. To test the delete functionality, add a new article and then click the "Release Article" button on the card.
